code/game/objects/machinery/squad_supply/supply_beacon.dm

```diff
@@ -57,6 +57,16 @@
 	playsound(src, 'sound/machines/twobeep.ogg', 15, 1)
 	H.visible_message("[H] activates [src].",
 	"You activate [src].")
+	var/marker_flags
+	if(H.faction == FACTION_TERRAGOV)
```

Comment:
remove_marker?

Reply:
not needed on this one remove marker is on deactivate of supply beacon